pallets/ddc-customers/Cargo.toml

```diff
@@ -11,10 +11,15 @@ frame-support = { version = "4.0.0-dev", default-features = false, git = "https:
 frame-system = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.30" }
 log = { version = "0.4.17", default-features = false }
 scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
+sp-io = { version = "6.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.30" }
```

Comment:
why do you need this? there is no changes to lib.rs so it must be related to mock/tests?

can this be moved to dev-dependencies?

Reply:
done